Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/TableView.cpp

```diff
@@ -143,6 +150,11 @@ namespace AzQtComponents
         handleResize();
     }
 
+    void TableView::mousePressEvent(QMouseEvent* event)
+    {
+        QTreeView::mousePressEvent(event);
+    }
+
```

Comment:
Is this override necessary? Is it bypassing something?

Reply:
Looks like `TableView` derives directly from `QTreeView`, so this can be removed.

Reply:
Removed.